OpenDreamClient/Interface/Html/HtmlParser.cs

```diff
@@ -139,6 +140,39 @@ void PushCurrentText() {
             appendTo.Pop();
     }
 
+    /**
+     * <summary>
+     * Returns if a tag is written in old self-closing form, or if the tag
+     * represents a void element, which must have no children
+     * </summary>
+     */
+    private static bool IsSelfClosing(string tagType, string[] attributes) {
+        if (attributes[^1] == "/") {
+            return true;
+        }
+
+        switch (tagType) {
```

Comment:
According to [this page of the DM reference](https://www.byond.com/docs/ref/#/DM/text/tags), the only tags in here that the client supports are `<br>` and `<img>`. Though it doesn't hint at this format of `<img>` being supported, so I'm not sure if that should be here either.

Reply:
Though it's good to be able to parse these, so this is fine.